lib/components/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "./src",
+    "types": [
```

Comment:
@kroeder  I had to use `types` to includes types of some library as it's done like this in "parent" [`tsconfig.json`](https://github.com/storybooks/storybook/blob/next/tsconfig.json), is there a reason to do it instead of letting TS compiler use all types available in `/node_modules/@types/` based on _@types, typeRoots and types_ of https://www.typescriptlang.org/docs/handbook/tsconfig-json.html